### PR TITLE
No GET support on collection endpoints

### PR DESCRIPTION
**Pull request by mt-clearhaus**
We are soon removing GET support for the account-wide collections, i.e. /authorizations, /captures, /refunds, /voids, /credits and /cards. Adjust the API documentation accordingly.

**Update (May 20):** All collection functionality is removed, not only account-wide.

@ct-clearhaus @soltveit RFR

### CODE REVIEW

File: source/index.md [hunk 1]
```diff
@@ -731,21 +731,21 @@ Resource collections are automatically paginated with most recent resource appea
...
-GET https://gateway.clearhaus.com/refunds?page=2&per_page=10
+GET https://gateway.clearhaus.com/authorizations/:id/captures?page=2&per_page=10
 ````
 
 Example response:
 
 ````json
 {
-    "count": 876,
+    "count": 23,
```
Comment (ct-clearhaus) [inline source/index.md]: 23 captures for one auth is still a bit much :smile:

**Comment by mt-clearhaus** [comment]
See https://github.com/clearhaus/issues-pci/issues/214#issuecomment-220321619 .

**Comment by soltveit** [comment]
👍

**Comment by mt-clearhaus** [comment]
@ct-clearhaus :eyes:

**Comment by ct-clearhaus** [comment]
:+1:

**Comment by mt-clearhaus** [comment]
@soltveit Feel free to deploy :-)

**Comment by soltveit** [comment]
Deployed.

**Pull request merged**